### ISSUE TITLE
Music player: can't use shuffle

### ISSUE THREAD

**Issue by DIKINGVFX** [BLeeEZ/amperfy#211]
Hi @BLeeEZ, 
Thank you for this amazing app ! Love it so far.

Though, I got a few issues or features that I would love to see :

### **1. When going to the songs tab, I can't use the shuffle option nor play in order.**

 _**How to reproduce it**_ : 
From Library go to Songs and then select any songs. 

--> Now you can't skip to the next songs nor the previous one. 
When going to the queue you find out that no songs are in the queue which is strange ? 
Same goes when using the shuffle option, no songs are added ?

**However**, instead of playing a random song, clicking on shuffle actually works, and changing between shuffle or play in order works ! 

### **2. When going to the Albums tab, I can't use the shuffle option nor play in order.**

 _**How to reproduce it**_ : 
From Library go to Songs and then select any albums.
As the previous point, it does exactly the same thing. I can't skip to the next album nor shuffle and so on.

### **3. I would love to have an easy way to create playlist.**

Basically, I would love to see an easy way to create playlists and add for instance albums to it rather than having to one by one going on each album to add to the playlist. 
I think an another option such as "select" in the 3 little dot in top right hand corner would be awesome ! 

Thanks again for your amazing works !

**Comment by BLeeEZ** [comment]
Hi,
thank you.

**1./2.** This is the current intended behaviour. The tables are displayed directly via database requests. Songs/Albums are huge data sets. Finding the previous/next entries in the displayed rows is complex and time-consuming. This is why Songs/Albums only play the selected element.

**3.** This sound linke a great feature. I need to do some investigation how realise this in the current UI.

**Comment by DIKINGVFX** [comment]
Hi, 
Thank you for your answer ! 

Yet, I still have a question about the **1./2.** 
How can I for instance choose a song from an album or choose a song from the songs tab, and they play the rest of my songs randomly ? 

Thank you again for your work, and it would be awesome to have the **3.** feature !

**Comment by noranraskin** [comment]
The native ios music app handles it in such a way that you get an alternative view of your music library and you can browse and search songs, albums, artists and playlists but every song you encounter has a round check/select button on the right and clicking the song adds/removes the song from the playlist until you hit the submit button on the top and you see the newly created playlist and I think that is an intuitive way to do it. Would love this feature
